aspnetcore/blazor/components/integration.md

```diff
@@ -186,9 +208,162 @@ Add a `Counter` component to the app with the interactive server render mode.
+
+After deciding which render mode to adopt:
+
+* If you plan to adopt the Auto render mode, follow the guidance in the [Enable interactive server rendering](#enable-interactive-server-rendering) section. 
+* If you plan to only adopt interactive WebAssembly rendering, continue without adding interactive server rendering.
+
+Add a package reference for the [`Microsoft.AspNetCore.Components.WebAssembly.Server`](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.Server) NuGet package to the app.
+
+[!INCLUDE[](~/includes/package-reference.md)]
+
+<!-- UPDATE 8.0 'Interactivity type' will change to 'Interactive render mode' at RTM -->
+
+Create a donor Blazor Web App to provide assets to the app. Follow the guidance in the <xref:blazor/tooling> article, selecting support for the following template features when generating the Blazor Web App.
+
+For the app's name, use the same name as the ASP.NET Core app, which results in matching app name markup in components and matching namespaces in code. Using the same name/namespace isn't strictly required, as namespaces can be adjusted after assets are moved from the donor app to the ASP.NET Core app. However, time is saved by matching the namespaces at the outset.
+
+Visual Studio:
+
+* For **Interactivity type**, select **Auto (Server and WebAssembly)**.
+* Set the **Interactivity location** to **Per page/component**.
+* Deselect the checkbox for **Include sample pages**.
+
+.NET CLI:
+
+* Use the `-int Auto` option.
+* Do ***not*** use the `-ai|--all-interactive` option.
+* Pass the `-e|--empty` option.
+
+From the donor Blazor Web App, copy the entire `.Client` project into the solution folder of the ASP.NET Core app.
+
+> [!IMPORTANT]
+> **Don't copy the `.Client` folder into the ASP.NET Core project's folder.** The best approach for organizing .NET solutions is to place each project of the solution into its own folder inside of a top-level solution folder. If a solution folder above the ASP.NET Core project's folder doesn't exist, create one. Next, copy the `.Client` project's folder from the donor Blazor Web App into the solution folder. The final project folder structure should have the following layout:
+>
+> * `AspNetCoreAppSolution` (top-level solution folder)
+>   * `AspNetCoreApp` (original ASP.NET Core project)
+>   * `AspNetCoreApp.Client` (`.Client` project folder from the donor Blazor Web App)
+>
+> For the ASP.NET Core solution file, you can leave it in the ASP.NET Core project's folder. Alternatively, you can move the solution file or create a new one in the top-level solution folder as long as the project references correctly point to the project files (`.csproj`) of the two projects in the solution folder.
+
+From the donor Blazor Web App, copy the `Components` folder into the ASP.NET Core app's project folder.
+
+In the ASP.NET Core project's `Components/Pages` folder, delete the `Home` component (`Home.razor`).
+
```

Comment:
I don't see why these steps are necessary. Can they just be removed?

```suggestion
```

Reply:
The `Error` component has no value?

Reply:
The Error component is useful for setting up an error page in your ASP.NET Core app when using the exception handling middleware, but it's not required.